api/who/who.proto

```diff
@@ -4,12 +4,22 @@ package who;
 
 message Void {}
 
+// None of these responses are cacheable.
+// Clients must send a client ID header.
 service WhoService {
 
   rpc putDeviceToken(PutDeviceTokenRequest) returns (Void);
 
   // The client should not pass a precise location.
   rpc putLocation(PutLocationRequest) returns (Void);
+
+  // TODO: Add location-specific stats.
+}
+
+// These responses are publicly cacheable and clients must
+// not send a client ID header.
+service PublicWhoService {
```

Comment:
I'd prefer we keep things simple and just use one service for now. All rpcs should always be idempotent. Caching policies for individual rpcs will likely vary.

Reply:
Per discussion down to one api for both local and global stats, thus removing the utility of a publicly-cacheable API/service.